pom.xml

```diff
@@ -257,6 +257,11 @@
       <artifactId>google-collections</artifactId>
       <version>1.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
+      <version>4.2</version>
+    </dependency>
```

Comment:
Our ecs push test was failing to use the sns broker because of this missing dependency.